docker-compose.yml

```diff
@@ -72,3 +72,5 @@ services:
     shm_size: 2g
     links:
      - "web:olympia.dev"
+    extra_hosts:
+     - "olympia.dev:web"
```

Comment:
this is a weird, weird, pattern.  `links` is really the opposite of `extra_hosts`?

Reply:
oh, you removed it in a later commit, nevermind.